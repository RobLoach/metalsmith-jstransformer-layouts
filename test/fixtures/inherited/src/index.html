---
layout: layouts/_child.jade
---

<h1>Hello World!</h1>

<p>This is a test.</p>
