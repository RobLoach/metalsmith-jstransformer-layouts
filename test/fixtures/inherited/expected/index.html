<!DOCTYPE html>
<html lang="en">
  <head>
    <title>This is the parent</title>
  </head>
  <body><div class="contents-wrapper"><div class="contents">
<h1>Hello World!</h1>

<p>This is a test.</p>
</div></div></body>
</html>